src/main/resources/application.yaml

```diff
@@ -84,12 +84,15 @@ launchDarkly:
 logging:
   level:
     root: INFO
+    uk.gov.hmcts.reform.wataskmanagementapi.clients.*: DEBUG
```

Comment:
are we sure about turning ON debug mode in Prod?

Reply:
This is only for the feign clients same thing is enabled on task management, it allows us to see each call being made and it's response